### PR TITLE
LLVM/Clang 10

### PR DESCRIPTION
**Pull request by q66**
# needs rebuilding (unticked still needs testing)

- [x] libclc-git
- [x] mesa
- [x] llvm9
- [x] bpftrace
- [x] gnome-builder
- [x] juCi++
- [x] kdevelop
- [x] qt5
- [x] qtcreator
- [x] rtags
- [x] shiboken2
- [x] rdedup
- [x] codelite
- [x] bcc
- [x] ccls
- [x] clazy
- [x] afl (x86 only, need help)
- [x] beignet (x86 only, need hlep)
- [x] ispc (x86 only, need help)
- [x] llvm-libunwind
- [x] libcxx
- [x] libcxxabi
- [x] include-what-you-use

# no bump but test build anyway

- [x] hfsprogs
- [x] godot
- [x] firefox
- [x] firefox-esr
- [x] bat
- [x] blender
- [x] ardour
- [x] pijul
- [x] qv4l2
- [x] yosys

### CODE REVIEW

File: srcpkgs/llvm10/template [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+ https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}/clang-tools-extra-${version}.src.tar.xz
+ https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}/compiler-rt-${version}.src.tar.xz"
+checksum="
+ df83a44b3a9a71029049ec101fb0077ecbbdf5fe41e395215025779099a98fdf
+ dd1ffcb42ed033f5167089ec4c6ebe84fbca1db4a9eaebf5c614af09d89eb135
+ b9a0d7c576eeef05bc06d6e954938a01c5396cee1d1e985891e0b1cf16e3d708
+ 885b062b00e903df72631c5f98b9579ed1ed2790f74e5646b4234fa084eacb21
+ acdf8cf6574b40e6b1dabc93e76debb84a9feb6f22970126b04d4ba18b92911c
+ 6a7da64d3a0a7320577b68b9ca4933bdcab676e898b759850e827333c3282c75"
+disable_parallel_builds=yes
```
Comment (Johnnynator) [inline srcpkgs/llvm10/template:37]:
```suggestion
disable_parallel_build=yes
```

File: srcpkgs/llvm9/template [hunk 2]
```diff
@@ -20,42 +20,15 @@ short_desc="Low Level Virtual Machine"
...
-checksum="
- 00a1ee1f389f81e9979f3a640a01c431b3021de0d42278f6508391a2f0b81c9a
- 8a7b9fd795c31a3e3cba6ce1377a2ae5c67376d92888702ce27e26f0971beb09
- 86262bad3e2fd784ba8c5e2158d7aa36f12b85f2515e95bc81d65d75bb9b0c82
- 5778512b2e065c204010f88777d44b95250671103e434f9dc7363ab2e3804253
- b26fd72a78bd7db998a26270ec9ec6a01346651d88fa87b4b323e13049fb6f07
- c2bfab95c9986318318363d7f371a85a95e333bc0b34fbfa52edbd3f5e3a9077"
+distfiles="https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}/llvm-${version}.src.tar.xz"
+checksum="00a1ee1f389f81e9979f3a640a01c431b3021de0d42278f6508391a2f0b81c9a"
 disable_parallel_builds=yes
```
Comment (Johnnynator) [inline srcpkgs/llvm9/template:25]:
```suggestion
disable_parallel_build=yes
```

**Comment by q66** [comment]
yeah, that line never had any effect, i removed it

**Comment by pullmoll** [comment]
So `bpftrace` no longer supports 32 bit targets. Please rebase that one.

**Comment by q66** [comment]
the bpftrace change in this PR was done correctly from the start, but then @leahneukirchen did the update and you put your own check on top, so now it conflicts and will need another bump

**Pull request merged**